### PR TITLE
Adding SDK-created extensions support

### PR DESCRIPTION
**Pull request by ifielker**
Adding SDK-created extensions support

### CODE REVIEW

**Comment by ifielker** [comment]
Added extension instance id in the list
image

Changed message for non-SDK-created extensions
image

New message for SDK-created extensions
image

Message no longer specific to non-SDK-created extensions (now includes both)
image

File: src/components/Extensions/Details/DetailCard/Tabs/ParamList.tsx [hunk 1]
```diff
@@ -21,36 +21,51 @@ import { Callout } from '../../../../common/Callout';
...
-              >
-                <Button>Learn more</Button>
-              </a>
-            </div>
-          </Accordion>
+          {isDynamic ? dynamicFragment : staticFragment}
+          <div className={styles.learnButton}>
+            <a
+              href={DOCS_BASE + docSuffix}
+              target="_blank"
```
Comment (tjlav5) [inline src/components/Extensions/Details/DetailCard/Tabs/ParamList.tsx:62]: You can use the ExternalLink component here, also why a tab-index of -1?

File: src/components/Extensions/Details/DetailCard/Tabs/ParamList.tsx [hunk 2]
```diff
@@ -21,36 +21,51 @@ import { Callout } from '../../../../common/Callout';
...
+              rel="noopener noreferrer"
+              tabIndex={-1}
+            >
+              <Button>Learn more</Button>
```
Comment (tjlav5) [inline src/components/Extensions/Details/DetailCard/Tabs/ParamList.tsx:66]: Hmm I don't think we should have buttons inside of anchors... that's not valid HTML

**Review by tjlav5 — APPROVED** [review]

File: src/components/Extensions/Details/DetailCard/Tabs/ParamList.tsx (hunk 1 above)
Comment (ifielker) [reply src/components/Extensions/Details/DetailCard/Tabs/ParamList.tsx:62]: discussed offline. This is ok.

**Pull request merged**